r128gain/__init__.py

```diff
@@ -578,6 +586,17 @@ def cl_main():
                                   Opus decoders so this should improve compatibility with players not supporting the
                                   R128 tags.
                                   Warning: This is EXPERIMENTAL, only use if you fully understand the implications.""")
+  arg_parser.add_argument("-p",
+                          "--preserve-times",
+                          action="store_true",
+                          help="Preserve modification times of tagged files")
+  arg_parser.add_argument("-P",
```

Comment:
I'm not convinced this second option is useful.

However assuming you will convince me, purely from a code standpoint: I think its better to have a single parameter with argparse aguments `default=False, nargs="?", const=0` (see https://docs.python.org/3/library/argparse.html?highlight=argparse#nargs).
That way `-p` implies `-p 0`, but you can also use `-p 3`, etc.

Reply:
Absolutely. I've never used argparse before so I didn't know this was possible.